README.md

```diff
@@ -501,6 +501,57 @@ The IP configuration supports the following options:
     Note that Classless inter-domain routing (CIDR) notation or network mask notation
     are not supported yet.
 
+- `routing_rule`
+
+    The policy routing rules can be specified via a list of rules given in the
+    `routing_rule` option, which allow routing the packets on other packet fields
+    except for destination address. The default value is a an empty list. Each rule is
+    a dictionary with the following entries:
+   - `priority` -
+      The priority of the rule. A valid priority ranges from 0 to 4294967295. Higher
+      number means lower priority.
+   - `action` -
+      The action of the rule. The possible values are `to-table` (default),
+      `blackhole`, `prohibit`, `unreachable`.
+   - `dport`-
+      The range of the destination port. The range consists of start, hyphen and end. A
+      valid dport value for both start and end ranges from 0 to 65534. And the start
+      cannot be greater than the end.
```

Comment:
It seems that I was not clear initially. Using the dollar sign seemed odd to me. For IP addresses, you provide an explizit example, so that would be better here as well instead of the "start, hyphen, and end" description, IMHO. Also, does this always need to be a range or is a single value ok, too?

Reply:
Single value (int/string ) is also ok, but it will be normalized into the range internally, I am hesitating to mention the single value support in the readme because it might confuse the user IMHO.

Reply:
Is this better ?